vars/fetchCheriSDK.groovy

```diff
@@ -47,7 +47,7 @@ def call(Map args) {
         gitBranch = env.BRANCH_NAME
     }
     if (!params.llvmBranch) {
-        if (gitBranch == 'dev' || gitBranch == 'devel')
+        if (gitBranch == 'c18n' || gitBranch == 'caprevoke' || gitBranch == 'cocall' || gitBranch == 'cocalls' || gitBranch == 'coexecve' || gitBranch == 'dev' || gitBranch == 'devel')
```

Comment:
I do wonder a bit if this should be inverted so everything but main, release/* and releng/* use dev.

Reply:
This applies to all projects, not just CheriBSD, which use all manner of weird branch names, so no